buffer.go

```diff
@@ -91,6 +92,11 @@ func (r *recordBuffer) Add(m *router.Message) error {
 		return err
 	}
 
+	// We default to a uuid if the template didn't match.
+	if pKey == "" {
+		pKey = uuid.New()
```

Comment:
Yeah it's only in debug mode, i could make debug mode the default. 

`r.pKeyTmpl` is just the template, i have no way to check it's something you want, unless 1) trying to compile it, 2) executing it against router.Message and see the result.

If the template fails to compile, right now we will log it (and not just on debug mode): https://github.com/remind101/logspout-kinesis/blob/master/drainer.go#L19. We could crash the goroutine but not sure what good it does to us.